### DISCUSSION TITLE
Next/Previous page scrolling

### DISCUSSION THREAD

**Discussion by chapmanjacobd** [community/community#103089] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When clicking next it seems wrong that the page doesn't scroll back to the top after loading the next page items (ie. `window.scrollTo({ top: 0 })`) with regard to lists paging.

Due to the existing behavior I've accidentally skipped some pages a few times because I didn't notice the next page of items loading because they load so quickly.

**Comment by gil-son** [comment]
Can you give me more details?

So, try to do that:

    window.scrollTo({ top: 0, behavior: 'smooth' });

**Reply by ghost** [reply]
or try to set the html scrol-behaviour property to smooth.

**Reply by chapmanjacobd** [reply]
This product feedback is about GitHub Lists--not an query about ECMAScript. Of course the final implementation details are up to GitHub whether or not to use smooth scrolling, though I would leave it disabled for `prefers-reduced-motion` (but maybe the browser will do that anyway)

**Reply by ghost** [reply]
Sorry for that : )

**Reply by Boskaine** [reply]
- [ ] @chapmanjacobd

**Reply by Boskaine** [reply]
I am trying

**Reply by Boskaine** [reply]
Okay

**Comment by Boskaine** [comment]
![IMG_20240130_190551_451.jpg](https://github.com/community/community/assets/90656284/4f45fa34-3392-41e4-adf2-e871cd1cdd20)

![IMG_20240130_190550_726.jpg](https://github.com/community/community/assets/90656284/5a81e8ca-8b58-4372-ae0c-0478daf1df4b)

![IMG_20240130_164846_690.jpg](https://github.com/community/community/assets/90656284/6e392692-b626-4cfc-b3f3-cb81ed30852d)

**Reply by Boskaine** [reply]
Help
